### ISSUE TITLE
Command with multiple arguments does not work

### ISSUE THREAD

**Issue by Willovent** [siokas/denomander#16]
When a command is provide with more than one argument,  VALIDATION_ARG_NOT_FOUND is thrown

repro steps : 
``` 
import Denomander from "https://raw.githubusercontent.com/siokas/denomander/master/mod.ts";

const program = new Denomander({
  app_name: "My MY App",
  app_description: "My MY Description",
  app_version: "1.0.1",
});

program
  .command("token [email] [password]")
  .action((email: string, password : any) => {
    console.log(email, password);
  })
  .description("get bearer token");

try {
  program.parse(Deno.args);
} catch (error) {
  console.log(error);
}
```
```
deno run .\index.ts token email password
```
outputs _`Error: Argument not found in available commands!`_

**Comment by siokas** [comment]
Currently I parse only one argument but I am working on it to work with multiple args.

**Comment by siokas** [comment]
Fixed in 9d978ddef2cb547d0bf41c0cecfe9b3f6376452e!
